src/coupled_simulation.jl

```diff
@@ -0,0 +1,169 @@
+    point_space_ice = CC.Spaces.PointSpace(context, coord)
+
+    atmos_facespace = CC.Spaces.FaceFiniteDifferenceSpace(center_space_atm)
+    boundary_space = CC.Spaces.level(
+        atmos_facespace,
+        CC.Utilities.PlusHalf(CC.Spaces.nlevels(atmos_facespace) - 1),
+    )
+
+    stepping = (;
+        Δt_min=Float64(physical_values[:delta_t_min]),
+        timerange=(Float64(0.0), Float64(physical_values[:t_max])),
+        Δt_coupler=Float64(physical_values[:delta_t_cpl]),
+        odesolver=CTS.ExplicitAlgorithm(CTS.RK4()),
+        nsteps_atm=physical_values[:n_t_atm],
+        nsteps_oce=physical_values[:n_t_oce],
+        nsteps_ice=1,
+    )
+    if physical_values[:sin_field_atm]
+        coord_field_atm = map(x -> x.z, CC.Fields.coordinate_field(center_space_atm))
+        field_atm = parameters.T_atm_ini .* (1 .- sin.((coord_field_atm .- parent(coord_field_atm)[1]) ./ (parent(coord_field_atm)[end] .- parent(coord_field_atm)[1]) .* (pi / 50)))
+    else
+        field_atm = CC.Fields.ones(Float64, center_space_atm) .* parameters.T_atm_ini
+    end
+
+    if physical_values[:sin_field_oce]
+        coord_field_oce = map(x -> x.z, CC.Fields.coordinate_field(center_space_oce))
+        field_oce = 1 .+ sin.((coord_field_oce .- parent(coord_field_oce)[1]) ./ (parent(coord_field_oce)[end] .- parent(coord_field_oce)[1]) .* (pi / 50))
+        field_oce = parameters.T_oce_ini .* (field_oce .- (parent(field_oce)[end] - 1))
+
+    else
+        field_oce = CC.Fields.ones(Float64, center_space_oce) .* parameters.T_oce_ini
+    end
+
+    T_atm_0 = CC.Fields.FieldVector(
+        atm=field_atm,
+    )
+    T_oce_0 = CC.Fields.FieldVector(
+        oce=field_oce,
+    )
+    T_ice_0 = CC.Fields.FieldVector(
+        ice=CC.Fields.ones(Float64, point_space_ice) .* parameters.T_ice_ini,
+    )
+
+    if boundary_mapping == "cit"
+        time_points_oce_domain = CC.Domains.IntervalDomain(
+            CC.Geometry.ZPoint{Float64}(stepping.timerange[1] - (stepping.Δt_min / 2)),
+            CC.Geometry.ZPoint{Float64}(stepping.timerange[2] - (stepping.Δt_min / 2));
+            boundary_names=(:start, :end),
+        )
+        time_points_atm_domain = CC.Domains.IntervalDomain(
+            CC.Geometry.ZPoint{Float64}(stepping.timerange[1] - (stepping.Δt_min / 2)),
+            CC.Geometry.ZPoint{Float64}(stepping.timerange[2] - (stepping.Δt_min / 2));
+            boundary_names=(:start, :end),
+        )
+
+        mesh_time_oce = CC.Meshes.IntervalMesh(time_points_oce_domain, nelems=Int(stepping.timerange[2] / stepping.Δt_min + 1))
+        mesh_time_atm = CC.Meshes.IntervalMesh(time_points_atm_domain, nelems=Int(stepping.timerange[2] / stepping.Δt_min + 1))
+        space_time_oce = CC.Spaces.CenterFiniteDifferenceSpace(device, mesh_time_oce)
+        space_time_atm = CC.Spaces.CenterFiniteDifferenceSpace(device, mesh_time_atm)
+        T_sfc = parameters.T_oce_ini .* CC.Fields.ones(space_time_oce)
```

Comment:
Shouldn't the surface and air temperatures only exist on a single point in space (but all points in time), not the whole space grid? So something like the `boundary_space` x `mesh_time_atm`/`mesh_time_oce`?

Reply:
Fixed in [94523ec](https://github.com/valentinaschueller/clima-playground/pull/11/commits/94523ecbaabfa7266ec83babcf3f9cb809ea3703)